src/org/labkey/test/AssayAPITest.java

```diff
@@ -106,6 +111,40 @@ protected void importAssayAndRun(File assayPath, int pipelineCount, String assay
         assertTextPresent(textToCheck);
     }
 
+    @Test
+    public void testGpatAssayOverAPI() throws Exception
+    {
+        String assayName = "testGpatAssay";
+        String assayDescription = "generated for test purposes over remoteAPI";
+
+        Connection connection = createDefaultConnection(true);
+        GetProtocolCommand getProtocolCommand = new GetProtocolCommand("General");                      // gets a template from the server
+        ProtocolResponse getProtocolResponse = getProtocolCommand.execute(connection, getCurrentContainerPath());
+
+        Protocol newAssayProtocol = getProtocolResponse.getProtocol();
+        newAssayProtocol.setName(assayName)
+            .setDescription(assayDescription)
+            .allowQCStates(true)
```

Comment:
This should be calling the setter for qcEnabled right, `setQcEnabled(true)?`

Reply:
doh, yes